packages/hub/src/session.ts

```diff
@@ -123,3 +134,120 @@ export class Session implements SessionInterface {
     });
   }
 }
+
+type releaseHealthAttributes = {
+  environment?: string;
+  release: string;
+};
+
+/**
+ * @inheritdoc
+ */
+export class SessionFlusher implements SessionFlusherInterface {
```

Comment:
This will be very hard to follow. We already went through this with various `Something` interface + `Something` class, where you couldn't tell which is which. In v7 I used `Like` convention, which we already do in v6 for promises and dsn. `SessionFlusher implements SessionFlusherLike` and the interface should have that exact name instead of being renamed via import statements.